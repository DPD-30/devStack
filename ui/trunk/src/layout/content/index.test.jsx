import React from 'react'
import { renderToString } from 'react-dom/server'
import { mount } from 'enzyme'


import LayoutContent from 'layout/content/index.jsx'

jest.unmock('layout/content/index.jsx')


test('LayoutContent matches snapshot', () => {
	const comp = renderToString(<LayoutContent />)

	expect(comp).toMatchSnapshot()
})

test ('LayoutContent shows error if swagger fails', () => {
    const comp = mount(<LayoutContent />)

	comp.setState({ swagger: 'ERROR' })

	comp.update()

	expect(comp.text().indexOf('Unable to connect to the API!') >= 0).toBeTruthy()
})
