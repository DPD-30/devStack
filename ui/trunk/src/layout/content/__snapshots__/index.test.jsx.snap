// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutContent matches snapshot 1`] = `
<div class="content"
     data-reactroot
>
  <routes>
  </routes>
</div>
`;
